packages/turf-meta/index.js

```diff
@@ -32,64 +32,73 @@ function coordEach(geojson, callback, excludeWrapCoord) {
         wrapShrink = 0,
         currentIndex = 0,
         isGeometryCollection,
-        isFeatureCollection = geojson.type === 'FeatureCollection',
-        isFeature = geojson.type === 'Feature',
+        type = (geojson) ? geojson.type : null,
```

Comment:
Well since a `Geometry` can be passed to `coordEach`, we need to handle `null` somehow.

Maybe its' best to add the `null` handling at the very top of the function.

```js
if (geojson === null) return;
```
That way the rest of the code can remain unchanged.